Mathlib/Analysis/Seminorm.lean

```diff
@@ -672,18 +670,14 @@ theorem ball_finset_sup' (p : ι → Seminorm 𝕜 E) (s : Finset ι) (H : s.Non
   induction H using Finset.Nonempty.cons_induction with
   | singleton => simp
   | cons _ _ _ hs ih =>
-    rw [Finset.sup'_cons hs, Finset.inf'_cons hs, ball_sup]
-    -- Porting note: `rw` can't use `inf_eq_inter` here, but `simp` can?
```

Comment:
The reason is that `inf_eq_inter` talks about `(fun x y => x \inf y) = ...` instead of the easier to apply `∀ x y, (x \inf y = ...)`. Should we redefine `inf_eq_inter`?